sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -1864,13 +1846,14 @@ public virtual Response DecommissionModel(string modelId, CancellationToken canc
         /// For more samples, see <see href="https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/digitaltwins/Azure.DigitalTwins.Core/samples">our repo samples</see>.
         /// </para>
         /// </remarks>
-        /// <seealso href="https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models" />
+        /// <see href="https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models">Understand twin models in Azure Digital Twins.</see>
```

Comment:
This tag needs to be inside summary or remarks. I don't think it can stand alone, like `seealso`.